packages/jsii/lib/assembler.ts

```diff
@@ -624,6 +624,27 @@ export class Assembler implements Emitter {
             }
         });
 
+        // Check that no interface declares a member that's already declared
+        // in a base type (not allowed in C#).
```

Comment:
C# allows this. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/new-modifier

Reply:
Well this is the error I get:
```
@aws-cdk/aws-iam: IRoleProxy.cs(113,21): error CS0111: Type 'IRoleProxy' already defines a member called 'AddToPolicy' with the same parameter types [/tmp/jsii-pacmak-codec20iQw/Amazon.CDK.AWS.IAM/Amazon.CDK.AWS.IAM.csproj]
```

Reply:
Yeah I guess 'new' will do it. I don't get the point of it though, and our C# generator doesn't do the necessary analysis to add the modifier, so I think disallowing in jsii is the easiest  way forward.